winml/test/model/model_tests.cpp

```diff
@@ -0,0 +1,166 @@
+#include "testPch.h"
+#include "onnxruntime_cxx_api.h"
+#include "test/onnx/TestCase.h"
+#include "test/onnx/heap_buffer.h"
+#include "test/util/include/test/compare_ortvalue.h"
+#include "ort_value_helper.h"
+
+#ifndef BUILD_GOOGLE_TEST
+#error Must use googletest for value-parameterized tests
+#endif
+
+using namespace onnxruntime::test;
+using namespace winml;
+using namespace onnxruntime;
+
+namespace WinML {
+// Global needed to keep the actual ITestCase alive while the tests are going on. Only ITestCase* are used as test parameters.
+std::vector<std::unique_ptr<ITestCase>> ownedTests;
```

Comment:
Nit: Should have internal linkage

Reply:
This should be cleared at `TearDown`, right? As-is it's only pushing tests and never cleaning them up

Reply:
Agreed, I'm adding a TearDownTestSuite()